packages/core/useElementVisibility/index.ts

```diff
@@ -43,12 +44,19 @@ export function useElementVisibility(
     }
   }
 
-  tryOnMounted(testBounding)
+  const stop = watch(
+    () => unrefElement(element),
+    () => testBounding(),
+    { immediate: true, flush: 'post' },
+  )
 
   if (window) {
-    tryOnMounted(() => useEventListener(
-      () => resolveUnref(scrollTarget) || window, 'scroll', testBounding, { capture: false, passive: true }))
+    useEventListener(scrollTarget || window, 'scroll', testBounding, {
+      capture: false, passive: true,
+    })
   }
 
+  tryOnScopeDispose(() => stop())
```

Comment:
this is redundant, `watch` already did it.

Reply:
Oops, just updated.